src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -55,14 +53,15 @@
  * gatk GetPileupSummaries \
  *   -I tumor.bam \
  *   -V common_biallelic.vcf.gz \
+ *   -V common_biallelic.vcf.gz \
```

Comment:
Why -V twice?

Reply:
Whoops -- one is `-V` and one should be `-L`.  I have to admit this is kind of clumsy but it's not utterly redundant, at least.  For example, one might want to use some custom interval list (`-L`) of common sites but grab the allele frequencies from gnomAD (`-V`).

Fixed it and expanded the javadoc a bit.